services/payments/Makefile

```diff
@@ -16,3 +16,15 @@ openapi.json: .env-ignore ## produces openapi.json
 	source $<; \
 	set +o allexport; \
 	python3 -c "import json; from $(APP_PACKAGE_NAME).main import *; print( json.dumps(the_app.openapi(), indent=2) )" > $@
+
+
+# NOTE: Create using `ln -s path/to/osparc-config/repo.config .env-secret`
+external ?= .env-secret
```

Comment:
did you start to experiment with osparc-config repo?

Reply:
yes. I basically did what it is in the note, i.e. creating a link to the config file. Then I would switch different branches and this way point to different deploys